src/components/TextInputWithCurrencySymbol/types.ts

```diff
@@ -28,6 +29,12 @@ type TextInputWithCurrencySymbolProps = {
 
     /** Whether the currency symbol is pressable */
     isCurrencyPressable: boolean;
-};
+
+    /** Whether to hide the currency symbol */
+    hideCurrencySymbol?: boolean;
+
+    /** Extra symbol to display next to the currency symbol */
```

Comment:
Whether the currency symbol is used or not depends on the other param
```suggestion
    /** Extra symbol to display */
```